### ISSUE TITLE
Status page of status pages

### ISSUE THREAD

**Issue by JesseWebDotCom** [OneUptime/oneuptime#709]
Imagine a company with various departments (sales, marketing, back office, HR, etc) each having their own status page for stuff. What would be nice for the company is a single status page for the entire company.

I plan on doing this by having a company status page with a monitor for each departments status page (using website monitor and checking for words in the body).  This seems like a hack and would be a great built in feature (and on incident, link to the underlying status page instead of an incdident).

**Comment by simlarsen** [comment]
@JesseWebDotCom what we can do is let you create embedded versions of those status pages, and you can stick them all up in a HTML page.

**Comment by JesseWebDotCom** [comment]
Cool - as long as it looks and feels like a standard status page. It's also a useful homepage for the company to then link to underlying department status pages. For now, I'll go with my suggestion until you all have something to beta.

**Comment by simlarsen** [comment]
@JesseWebDotCom another idea - can you create a master status page which has groups and resources underneath. Would that work?

**Comment by JesseWebDotCom** [comment]
> @JesseWebDotCom another idea - can you create a master status page which has groups and resources underneath. Would that work? 

I don't think so as I don't care about HR's xyz or abc systems - I just want to know whether HR systems are overall good or not.

**Comment by JesseWebDotCom** [comment]
My suggestion won't work because the body I get back from monitoring a status page doesn't return a full body - it says things like "You need JavaScript to run this app" and "This HTML file is a template"

**Comment by JesseWebDotCom** [comment]
> @JesseWebDotCom another idea - can you create a master status page which has groups and resources underneath. Would that work?

And I can't use your suggestion as monitors are scoped to projects - so the xyzzy company project cannot see the monitors in the HR project.
